capsules/src/l3gd20.rs

```diff
@@ -174,6 +175,18 @@ enum L3gd20Status {
 //     Idle,
 // }
 
+pub struct App {
+    upcall: Upcall,
+}
+
+impl Default for App {
+    fn default() -> App {
+        App {
+            upcall: Upcall::default(),
+        }
+    }
+}
```

Comment:
This could be just a `#[derive(Default)]` annotation

Reply:
I think this is fine, in particular since #2462 does not include the proc-macro-based derive `GrantDefault` implementations, so this will make our lives slightly easier. :smile: